internal/commands/result.go

```diff
@@ -1099,7 +1108,7 @@ func CreateScanReport(
 	targetFile,
 	targetPath string,
 	agent string,
-	params map[string]string,
+	resultsParam map[string]string,
```

Comment:
it not consistent ( params, resultsParam, resultsParams) Lets have resultsParams